package.nix

```diff
@@ -41,8 +41,14 @@ stdenv.mkDerivation (finalAttrs: {
 
   src = lib.cleanSourceWith {
     src = ./.;
-    filter = path: type:
-      ! (lib.hasSuffix ".nix" path);
+    filter = (
+      path: type: (builtins.all (x: x != baseNameOf path) [
+        ".git"
```

Comment:
It is very problematic to add `.git` directory to a Nix package source. This directory is changing all the time which will cause Nix package to rebuild all the time.

Regarding `.github` directory, I added it to the filter to prevent Nix package rebuilds when CI files are changed.
`docker/testdata/test_docker_image.sh` is not used during the package build.

Reply:
Perfect then!